### PR TITLE
Add the 18.10 takeover

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Created the 18.10 takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that the takeover matches the [design](https://github.com/ubuntudesign/www.ubuntu.com-design/issues/159) and [copy doc](https://docs.google.com/document/d/18s-sTyuuYZP7xrpn7MtN5ABScfv0QVOCdRbEGZ44cg4/edit)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4037

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -18,11 +18,12 @@
       section lang="en" data-lang="en" class="p-strip is-deep p-takeover--robotics js-takeover {% if not default %}u-hide{% endif %}" {% if default %}data-default="true"{% endif %}
 
   - Also, add a 'with default=True' below for one takeover to be the default
-  
+
   #}
 
 {% block takeover_content %}
-    {% include "takeovers/_robotics-takeover.html" with default=True %}
+    {% include "takeovers/_1810-takeover.html" with default=True %}
```
Comment (caldav) [inline templates/index.html]: This should probably be the only takeover for a week.

**Comment by anasereijo** [comment]
@anthonydillon 

Animation:
- the bubbles are too subtle on some screens. Is it possible to make them more visible?
- can you make the animation continuous? there is no bubble for a few seconds - we want bubbles coming up all the time :)

mobile:
- can we center the cuttlefish and make it slightly smaller? see below my original design:
takeover-small-screen

- the cuttlefish looks cropped at the bottom. can you please fix it? see below 

screenshot 2018-10-03 at 09 41 56

**Comment by anthonydillon** [comment]
@anasereijo I have done what I can to tackle each issue. I have made the bubbles continuous and improved the small screen view. There are some viewports that get cropped as the original image is cropped on the artboard. Can you take another look please.

**Comment by anasereijo** [comment]
@anthonydillon here goes the cuttlefish with the amended artboard: https://assets.ubuntu.com/v1/de27c5e6-CUTTLEFISH_Mascot_for+web-WHT.svg

the animation and mobile version are looking much better, thanks! 

tablet is cropped:
screenshot 2018-10-03 at 12 30 59

**Comment by anthonydillon** [comment]
@anasereijo thanks, the new image is much better. Added and aligned nicely.

**Comment by anasereijo** [comment]
Hi @anthonydillon mobile and desktop are looking great 👍 but for some reason seems like the cuttlefish is smaller now on desktop... can you please make it bigger just on desktop? check my design bellow:

final - takeover-large-screen

**Pull request merged**